src/classlibnative/config/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
```

Comment:
Creating a new directory, makefile and lib to make for just a single function looks like overkill. Can't this be added to existing one?